blue_bringup/config/gazebo_controllers.xml

```diff
@@ -0,0 +1,205 @@
+<!--
```

Comment:
Would it be possible to move this to the blue_simulator repository?

Reply:
Sure thing!